src/components/exit-this-page/index.md

```diff
@@ -0,0 +1,122 @@
+
+### Choosing a different website to load
+
+The component is set to take the user to the [homepage of BBC Weather](https://www.bbc.co.uk/weather). You can change this if there’s a more appropriate site for your users.
+
+Avoid websites that might show personalised pages (such as frequently visited, last visited, or suggested links), as this content could put the user at risk.
+
+## Consider what to do user session data
+
+You’ll need to decide if your service will store the user’s session data after they’ve activated the Exit this page component and left your service.
+
+We recommend that services clear the user’s session data once they’ve activated Exit this page, whether they've done so by using the button, the secondary skip link or pressing the shift button 3 times.
+
+## Adding the secondary link
+
+The code will need to go in the layout file of your service. It will need to go after your block header.
+
+To activate the secondary link, users will need to either:
+- tab to the Exit this page link
+- press the shift key 3 times
+
+These will perform the same action as pressing the button.
+
+### Showing progress for 'Shift key' presses
+
+To show the user the number of times they’ve pressed the shift button, there’s a series of 3 progress dots under the button, provided as a visual enhancement. Each time the shift key is pressed, one of the dots will be filled.
+
+The user has 5 seconds to press the shift key 3 times. After that, the sequence and progress dots will reset.
+
+This feature requires the user to have Javascript enabled on their browser.
+
+## Loading overlay
+
+When the user activates Exit this page, a ‘loading overlay’ will immediately appear. This is to help users with slow connections clear their screen until the browser loads the next page.
+
+## Research on this component
+
+### Accessibility features and considerations
+
+In addition to the 'Exit this page' button itself, users have 2 other options to activate the component. We've tested and iterated both of these options to be accessible, reliable and discreet as possible for users.
+
+The keyboard option, where the user can press shift 3 times, gives users a discreet way to activate the component, particularly keyboard-only users. We considered options that used a single key press (such as the 'Esc' key), but we could not be confident that this would not interfere with other user actions.
+
+The 'skip link' option is a hidden link in the tab order of the page. This gives users of assistive technology and easier way to reliably activate the component.
+
+To help users of speech recognition software activate the 'skip link' more discreetly, we considered disguising the 'Exit this page' link text with something else. However, we decided that this could confuse users and other methods, such as referring to the link by item number would meet their need.
```

Comment:
I don't fully understand this bit. Was the 'something else' considered also the 'item number' mentioned later? Or is item number the chosen method? And also what methods are confused?

Reply:
```suggestion
To help users of speech recognition software activate the 'skip link' more discreetly, we considered  renaming the 'Exit this page' link into something else that hid its real purpose. 

However, we decided this was more likely to confuse users. We're also confident about the other ways users can use speech recognition software to activate the 'skip link' without saying the text out loud, such choosing the link as an item number on a list.
```